### PR TITLE
A few UI tweaks for New Show Planner

### PR DESCRIPTION
**Pull request by mstratford**
Also fixes the Multi Output audio selection in newer Chrome versions.

### CODE REVIEW

**Comment by LordAro** [comment]
Will do a full review later, but did you fix the "attempting to reorder search results causes error" issue? Don't see it explicitly mentioned in commits, is all

**Comment by mstratford** [comment]
@LordAro It was the first commit 👍

**Comment by mstratford** [comment]
Fixes #612

**Comment by mstratford** [comment]
Note, the length field in bapsplanner.managed_items will need to have the decimal points removed before this is updated.

**Comment by LordAro** [comment]
Why can you not format the time accordingly? the microsecond accuracy is built into postgres' column type and isn't all that easy to (permanently) remove

File: schema/data-actions.json [hunk 1]
```diff
@@ -71,7 +71,6 @@
     ["NIPSWeb", "secure_play"],
     ["NIPSWeb", "managed_play"],
     ["NIPSWeb", "load_aux_lib"],
-    ["NIPSWeb", "get_client_token"],
```
Comment (LordAro) [inline schema/data-actions.json]: Think this means something needs to be removed from the database?

File: src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php [hunk 2]
```diff
@@ -151,7 +148,7 @@ public function toDataSource($mixins = [])
             'summary' => $this->getTitle(), //Again, freaking NIPSWeb
             'title' => $this->getTitle(),
             'managedid' => $this->getID(),
-            'length' => CoreUtils::happyTime($this->getLength() > 0 ? $this->getLength() : 0, true, false),
+            'length' => $this->getLength(),
```
Comment (LordAro) [inline src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php]: an int() here would do the job on truncating the decimal points, right?

File: schema/data-actions.json (hunk 1 above)
Comment (mstratford) [inline schema/data-actions.json]: I guess so, I can do that :)

File: src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php (hunk 2 above)
Comment (mstratford) [inline src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php]: Well, it comes out as a hh:mm:ss.ms so I'm not sure int will work. However I will add a substring thing here.

**Comment by mstratford** [comment]
This doesn't fix / work around the DELETE issue, I don't know what's causing that in the URY environment.

**Comment by mstratford** [comment]
Removed the need for the SQL decimal point edit using the above commit.

File: src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php [hunk 3]
```diff
@@ -72,7 +72,7 @@ protected function __construct($resid, $playlistref = null)
                 );
         $this->folder = $result['folder'];
         $this->title = $result['title'];
-        $this->length = $result['length'];
+        $this->length = strtotime('1970-01-01 '.$result['length']. ' UTC');
```
Comment (LordAro) [inline src/Classes/NIPSWeb/NIPSWeb_ManagedItem.php]: i feel like there should be a timeToInt function somewhere...

File: schema/data-actions.json (hunk 1 above)
Comment (mstratford) [inline schema/data-actions.json]: Done

**Pull request merged**